src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,29 @@
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+
+const initialState = [];
+
+export const addBook = (book) => ({
+  type: ADD_BOOK,
+  book,
+});
+
+export const removeBook = (bookID) => ({
+  type: REMOVE_BOOK,
+  bookID,
+});
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, action.book];
+    case REMOVE_BOOK: {
+      const { bookID } = action;
+      return state.filter((book) => book.id !== bookID);
+    }
+    default:
+      return state;
+  }
+};
+
+export default reducer;
```

Comment:
- [ ] You need to import { createSlice } from "@reduxjs/toolkit" and use it to create a reduxer. Refer to this [document](https://redux-toolkit.js.org/api/createslice). You did something similar to categories slice

Example:
```
import { createSlice } from '@reduxjs/toolkit'
import type { PayloadAction } from '@reduxjs/toolkit'

interface CounterState {
  value: number
}

const initialState = { value: 0 } as CounterState

const counterSlice = createSlice({
  name: 'counter',
  initialState,
  reducers: {
    increment(state) {
      state.value++
    },
    decrement(state) {
      state.value--
    },
    incrementByAmount(state, action: PayloadAction<number>) {
      state.value += action.payload
    },
  },
})

export const { increment, decrement, incrementByAmount } = counterSlice.actions
export default counterSlice.reducer
```